e2e/setup.go

```diff
@@ -180,7 +179,10 @@ func TestSuiteConfig(t *testing.T, testCfg *Cfg) server.CLIConfig {
 	}
 
 	maxBlobLengthBytes, err := utils.ParseBytesAmount("16mib")
-	require.NoError(t, err)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
same here, why change to the wordier if/panic?

Reply:
because we can longer pass `t` through via the constructor using this new pattern type